gravitee-apim-portal-webui-next/src/app/registration/registration.component.html

```diff
@@ -0,0 +1,118 @@
+
+    Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+@if (!submitted) {
+  <form
+    [formGroup]="registrationForm"
+    (ngSubmit)="register()"
+    class="registration__form__container"
+    [appIsMobile]="'registration__form__container--mobile'">
+    <mat-card appearance="outlined" class="registrationn">
+      <mat-card-header class="registration__form__header">
+        <mat-card-title class="registration__form__title" i18n="@@registrationTitle">CREATE AN ACCOUNT</mat-card-title>
+      </mat-card-header>
+      <mat-card-content class="registration__form__content">
+        <div class="registration__form">
+          <div class="registration__form__fields">
+            <mat-form-field appearance="outline" class="registration__form__input">
+              <mat-label i18n="@@registrationFirsname">First name</mat-label>
+              <input matInput formControlName="firstname" />
+              @if (registrationForm.controls.firstname.hasError('required')) {
+                <mat-error i18n="@@registrationFirsnameErrorRequired">First name is required</mat-error>
+              }
+            </mat-form-field>
+            <mat-form-field appearance="outline" class="registration__form__input">
+              <mat-label i18n="@@registrationLastname">Last name</mat-label>
+              <input matInput formControlName="lastname" />
+              @if (registrationForm.controls.lastname.hasError('required')) {
+                <mat-error i18n="@@registrationLastnameErrorRequired">Last name is required</mat-error>
+              }
+            </mat-form-field>
+            <mat-form-field appearance="outline" class="registration__form__input">
+              <mat-label i18n="@@registrationEmail">Email</mat-label>
+              <input matInput formControlName="email" />
+              @if (registrationForm.controls.email.hasError('required')) {
+                <mat-error i18n="@@registrationEmailErrorRequired">Email is required</mat-error>
+              }
+              @if (registrationForm.controls.email.hasError('email')) {
+                <mat-error i18n="@@registrationEamilErrorFormat">Email format error</mat-error>
+              }
+            </mat-form-field>
+
+            @if ((customUserFields ?? []).length > 0) {
+              <div formGroupName="customFields">
+                @for (customField of customUserFields; track customField.key) {
```

Comment:
Related to https://github.com/gravitee-io/gravitee-api-management/pull/14707/changes#r2640097808

Retrieve value of a signal

```suggestion
            @if ((customUserFields() ?? []).length > 0) {
              <div formGroupName="customFields">
                @for (customField of customUserFields(); track customField.key) {